lib/pacemaker/pcmk_output.c

```diff
@@ -1999,6 +2002,21 @@ cluster_status_xml(pcmk__output_t *out, va_list args)
                      false);
     }
 
+
+    /* If there are verification errors, always print a statement about that, even if not requested */
+
+    pcmk__output_new(&verify_out, "none", NULL, NULL);
+    verify_rc = pcmk__verify(scheduler, verify_out, scheduler->input);
+    pcmk__output_free(verify_out);
```

Comment:
You also need to set `scheduler->priv = verify_out` before calling `pcmk__verify()` (and save and restore the original value) because the scheduler code will output messages there too (`pcmk__output_cluster_status()` sets `scheduler->priv` before calling the cluster-status message)